Website/Providers/DataProviders/SqlDataProvider/DotNetNuke.Schema.SqlDataProvider

```diff
@@ -19496,7 +19496,7 @@ As
 	ELSE
 		BEGIN
 			-- AssemblyID is not null - Assembly is registered - test for version
-			SET @CurrentVersion  = (SELECT Version 
+			SET @CurrentVersion  = (SELECT Max(Version ) 
```

Comment:
@galatrash-at-dnn I've pushed a change which properly uses a function to parse the version into integers and then sorts based on that.  Does that look better?